dev/io.openliberty.microprofile.config.2.0.internal_fat/publish/servers/Config20Server/bootstrap.properties

```diff
@@ -4,3 +4,5 @@ io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnam
 io.openliberty.microprofile.config.internal_fat.apps.brokenInjection.ConfigUnnamedMethodInjectionBean.SIMPLE_KEY6=VALUE6
 noConverterKey=value
 org.jboss.logging.provider=slf4j
+value1DefinedInBootstrapProperties=value1
+value2DefinedInBootstrapProperties=value2
```

Comment:
Can you define the one in `microprofile-config.properties` instead?

Reply:
Good idea. I'll give that a try :)

Reply:
How does this look: https://github.com/OpenLiberty/open-liberty/pull/14823/commits/c3296348f6cd8ce48a0f24b7db82c38f4b4b7651 ?